src/constants.h

```diff
@@ -97,6 +97,28 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 // Default maximal breath of a player
 #define PLAYER_MAX_BREATH_DEFAULT 10
 
+
+// DEFAULT_PHYSICS constants of a player
```

Comment:
These C++ changes should be removed; this is dead code that doesn't have any effect on anything at all.

Reply:
You mean these definations such as `#define PLAYER_SPEED_DEFAULT `?

Reply:
Yes